modules/vulnerability-management-accept-risks.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_module-type: PROCEDURE
+[id="vulnerability-management-accept-risks_{context}"]
+= Accepting risks
+
+.Prerequisites
+* You must have the `write` permission for the `VulnerabilityManagementRequests` resource.
+
+To accept risk with or without mitigation:
+
+.Procedure
+. On the RHACS portal, navigate to *Vulnerability Management* -> *Dashboard*.
+. On the *Dashboard* view header, select *Images*.
+. From the list of images, select the image you already assessed.
+. Find the row which lists the CVE you would like to take action on.
+. Click the three dots icon on the right for the CVE you identified and click *Defer CVE*.
```

Comment:
```suggestion
. Click {kebab} on the right for the CVE you identified and click *Defer CVE*.
```

Should we use the kebab icon instead of "three dots icon" ? 
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#inline-images-icons